task/sinker.go

```diff
@@ -453,6 +455,21 @@ func (s *Sinker) applyAnotherConfig(newCfg *config.Config) (err error) {
 	return
 }
 
+func applyCredentials(newCfg *config.Config, cred util.Credentials) {
+	if cred.ClickhouseUsername != "" {
+		newCfg.Clickhouse.Username = cred.ClickhouseUsername
+	}
+	if cred.ClickhousePassword != "" {
+		newCfg.Clickhouse.Password = cred.ClickhousePassword
+	}
+	if cred.KafkaUsername != "" {
```

Comment:
How to provide `Config.Kafka.Sasl.GSSAPI.Username` and password?

Reply:
Add two new envs for GSSAPI